### ISSUE TITLE
[Bug] xml tag rainbow colouring breaks when opening tag contains wrong whitespace

### ISSUE THREAD

**Issue by PlasmaHH** [luochen1990/rainbow#195]
### **Describe the bug**
When an xml opening tag contains an attribute and after that there is some whitespace, then the rainbow colouring does not work

### **To Reproduce**

```xml
<a foo="" >
<b>
</b>
</a>
```

or 

```xml
<a foo="" 
>
<b>
</b>
</a>
```

For the first case replaying  `\>/`  with `\s*\>/`  in the regex for start= works, but for the second there is probably some way that it needs to match over multiple lines

**Comment by luochen1990** [comment]
@PlasmaHH The default config is fixed now, can you help to test it?

**Comment by PlasmaHH** [comment]
> @PlasmaHH The default config is fixed now, can you help to test it?

Yes, it works for html. Note that my initial report was about xml where it should be added too and I guess as well in xhtml too
